csv_export_payment/models/csv_export_payment.py

```diff
@@ -27,12 +27,15 @@ class PartnerCSVExport(models.TransientModel):
     def get_domain(self):
         # est-ce que les paiments peuvent rester en draft plusieurs jours?
         #  => non, les paiements ne passent pas par l'état brouillon.
```

Comment:
I know it's not your comment but can you translate this ?

Reply:
done